src/fontra/client/core/actions.js

```diff
@@ -27,19 +28,22 @@ actionInfoController.addListener((event) => {
   actionsByKeyOrCode = undefined;
 });
 
+let sortIndex = 0;
 export function registerAction(actionIdentifier, actionInfo, callback, enabled = null) {
-  registerActionInfo(actionIdentifier, actionInfo);
+  registerActionInfo(actionIdentifier, actionInfo, sortIndex);
   registerActionCallbacks(actionIdentifier, callback, enabled);
+  sortIndex++;
```

Comment:
I think the sortIndex logic should be done by registerActionInfo() itself. And it would not need the sortIndex argument.

Reply:
Perhaps each topic should have its own sortIndex counter.

I assume the topic in the UI are currently also sorted based on the sortIndex?

Reply:
I moved the sortIndex logic out of registerAction into registerActionInfo.

Reply:
As discussed, I moved the sorting out of actions into the UI code (shortcuts panel) – including the discussed refinements to topic indices (00X0-action-topics.XXX) etc.